### PR TITLE
Support Brownian dynamics in LAMMPS

### LINKED ISSUE #166: Support Brownian dynamics in LAMMPS
Brownian dynamics was added in a more recent release. We should identify the first stable version that had the bug fix I added to ensure the friction coefficients were set correctly, then support this operation for LAMMPS. We can do a version check to make sure LAMMPS is compatible, and error out at call time if it is not.

### PR DESCRIPTION
**Pull request by mphoward**
This PR adds conditional / experimental support for Brownian dynamics in LAMMPS. There are several checks needed to make sure LAMMPS can run this type of simulation, and the CI does not currently include the `BROWNIAN` package so it is not very well tested. However, we can probably test in production since the changes are not too complicated.

This should be rebased and merged after PR #170.

Fixes #166

### CODE REVIEW

**Comment by mphoward** [comment]
Noting: we should parse the packages that are installed with LAMMPS for `BROWNIAN` to make sure this command can succeed.

**Comment by mphoward** [comment]
This is ready for review.

File: src/relentless/simulate/lammps.py [hunk 1]
```diff
@@ -528,6 +528,95 @@ def _make_T(thermostat):
             return (thermostat.T, thermostat.T)
 
 
+class RunBrownianDynamics(_Integrator):
+    """Perform a Langevin dynamics simulation.
```
Comment (clpetix) [inline src/relentless/simulate/lammps.py:532]: Should this be Brownian dynamics?

**Review by clpetix — APPROVED** [review]
Besides the small issue in the documentation, this all looks good to me!

File: src/relentless/simulate/lammps.py (hunk 1 above)
Comment (mphoward) [reply src/relentless/simulate/lammps.py]: Yep!

**Pull request merged**